css/main.scss

```diff
@@ -49,5 +47,6 @@ $on-laptop:        800px;
 @import
         "base",
         "layout",
-        "syntax-highlighting"
+        "syntax-highlighting",
+        "kakapo"
```

Comment:
also generic name would be better, like `documentation`since we will use it for other projects hopefully